### PR TITLE
Update README and catalog-info.yaml per maintainers pilot.

### PR DESCRIPTION
**Pull request by bradenmacdonald**
## Description

This updates the README and adds `catalog-info.yaml` file per the requirements of OEP-55 and the [Maintainership Pilot](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3426844690/Maintainership+Pilot)

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @bradenmacdonald!

As a core committer in this repo, you can merge this once the pull request is approved per the [core committer reviewer requirements](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1529675973/Open+edX+Core+Committers#Reviews-and-Ownership) and according to the agreement with your edX Champion.

**Review by Agrendalath — APPROVED** [review]
:+1:

* [x] I tested this: read the docs, checked the `catalog-info.yaml`
* [x] I read through the code
* [x] I checked for accessibility issues: n/a
* [x] Includes documentation
* [x] I made sure any change in configuration variables is reflected in the corresponding client's `configuration-secure` repository: n/a

File: README.rst
Comment (bradenmacdonald) [reply README.rst:5]: I copied it from another example, but I see now it's not in [the official cookiecutter template](https://raw.githubusercontent.com/openedx/edx-cookiecutters/master/python-template/%7B%7Bcookiecutter.placeholder_repo_name%7D%7D/README.rst) so I'll remove it.

**Comment by openedx-webhooks** [comment]
@jristau1984: thought you might like to know that bradenmacdonald merged this pull request.

**Comment by openedx-webhooks** [comment]
@bradenmacdonald 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/blockstore&entry.867055334=https://github.com/openedx/blockstore/pull/207&entry.1484655318=https://github.com/bradenmacdonald&entry.752974735=2022-10-12+17:49&entry.1917517419=2022-10-17+17:03&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**
